released/packages/coq-flocq/coq-flocq.3.2.0+8.11/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+maintainer: "guillaume.melquiond@inria.fr"
+homepage: "http://flocq.gforge.inria.fr/"
+dev-repo: "git+https://gitlab.inria.fr/flocq/flocq.git"
+bug-reports: "https://gitlab.inria.fr/flocq/flocq/issues"
+license: "LGPL 3"
+build: [
+  ["./autogen.sh"]
+  ["./configure"]
+  ["./remake" "-j%{jobs}%"]
+]
+install: ["./remake" "install"]
+depends: [
+  "coq" {= "8.11.0"}
+]
+tags: [ "keyword:floating point arithmetic" ]
```

Comment:
```suggestion
tags: [ "keyword:floating point arithmetic" "logpath:Flocq" ]
```

Reply:
Can you please quickly explain the intention of the logpath tag (or post a link)?

Reply:
`logpath` is how we keep track of which Coq namespace a package uses, i.e., the name of the directory in `user-contrib`. If two packages have the same `logpath`, they must not be installable at the same time, since otherwise files will be overwritten by OPAM. See https://github.com/coq/ceps/blob/master/text/003-opam-metadata.md#logpath

Reply:
as useful feature - thanks for the explanation!